ITISFoundation/osparc-simcore
# 🐛 Fix 'make down'

## What do these changes do?

Some changes to prevent the following three messages from occurring during the running of 'make down' in the root of the repo. (Especially if one runs make down several times consecutively) 

1)
`# Removing client containers (if any)
[+] Running 1/0
 ! Network tools_default  Resource is still in use `

2)
`# Removing generated docker compose configurations, i.e. .stack-*
rm: missing operand
Try 'rm --help' for more information.
make: [Makefile:396: down] Error 1 (ignored)`

3)
`# Removing local registry if any
Error response from daemon: No such container: registry`

## Related issue/s




## How to test

Run 'make down' after 'make up-devel', even several times in a row. No error messages should be generated.

## DevOps Checklist

N/A
